src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Commands.AzureStackStorage.csproj

```diff
@@ -214,6 +212,10 @@
       <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
       <Name>Commands.ResourceManager.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Resources\Commands.Resources\Commands.Resources.csproj">
```

Comment:
You should not add this reference.  This brings over all of the additional unneeded files you have in the .wxi file.  It also puts a tight coupling between these two modules.  Any needed methods from resource manager should go into common code, or directly into this module